reference/configuration/security.rst

```diff
@@ -1075,6 +1075,58 @@ the session must not be used when authenticating users:
             // ...
         };
 
+.. _reference-security-lazy:
```

Comment:
Is that used somewhere or mandatory for any reason ?

Reply:
a simple copy/paste from "stateless" doc part for now :s